Source/WebInspectorUI/UserInterface/Models/CSSKeywordCompletions.js

```diff
@@ -230,6 +229,8 @@ WI.CSSKeywordCompletions.forFunction = function(functionName)
         suggestions.pushAll(WI.CSSKeywordCompletions._colors);
     }
 
+    suggestions.pushAll(additionalFunctionValueCompletionsProvider?.(functionName) ?? []);
```

Comment:
NIT: I think we could/should check `additionalFunctionValueCompletionsProvider` before attempting to call `pushAll`

```
if (additionalFunctionValueCompletionsProvider)
    suggestions.pushAll(additionalFunctionValueCompletionsProvider(functionName));
```

no reason to call `pushAll` with an empty array :)

Reply:
Done